.github/workflows/build.yml

```diff
@@ -8,7 +8,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        options: [ '-mb braces', '--ie8 -c', '-mc', '--toplevel -mc passes=3,pure_getters,unsafe' ]
+        options:
+          - '-p acorn -mb braces -o spidermonkey'
```

Comment:
@kzc so what I did is to augment `--in-situ` to handle `-o spidermonkey` by doing a round-trip, i.e. `AST_Node.from_mozilla_ast(uglified.to_mozilla_ast())`, which should cover this beyond `.ts` files which you've done in https://github.com/mishoo/UglifyJS/pull/4870#issuecomment-826454643

Reply:
I forgot about `-p acorn`.

Reply:
You weren't alone there 😉

Reply:
So does `--in-situ` unconditionally round-trip uglify -> mozilla -> uglify AST now?

Reply:
Not with `--in-situ` alone &minus; you need to specify `--output spidermonkey`

Reply:
I see. Okay.